docs/Tutorials/OpenAPI.md

```diff
@@ -304,6 +401,97 @@ the JSON response payload defined is as follows:
 ]
 ```
 
+
+### Examples
+
+To define a reusable example definition you can use the [`Add-PodeOAComponentExample`](../../Functions/OpenApi/Add-PodeOAComponentExample) function. You'll need to supply a Name, a Summary and a list of value representing the object.
+
+The following is an example that define three Pet examples and it's used update pet operation
```

Comment:
Grammar fix: `The following is an example that defines three Pet examples request bodies, and how they're used in a Route's OpenAPI definition:`

Reply:
I'm stilling seeing this one.